### DISCUSSION TITLE
Dreambooth on Runpod error

### DISCUSSION THREAD

**Discussion by methodicalculator** [TheLastBen/fast-stable-diffusion#2624] [Q&A] [answered]
Hi! I'm kinda new using Runpod so excuse me if it's a stupid question. I'm trying to modeling with Dreambooth but when I try to download a model I got stuck with this error.
![Schermata 2023-10-23 alle 01 30 41](https://github.com/TheLastBen/fast-stable-diffusion/assets/127632592/d985e1bc-6aa0-444f-8fec-4fa3ac426760)

**Comment by TheLastBen** [comment, marked as answer]
Use the correct format for the link, remove anything after "?"

**Reply by methodicalculator** [reply]
It was even more dumb than I thought! Now I have the problem that Dreambooth can't convert the model I've downloaded.

It appears this error:
rm: cannot remove 'convertodiffv2.py': No such file or directory
